src/bp/core/services/hook/hook-service.ts

```diff
@@ -179,10 +179,12 @@ export class HookService {
     clearRequireCache()
   }
 
-  async executeHook(hook: Hooks.BaseHook): Promise<void> {
+  async executeHook(hook: Hooks.BaseHook, throwOnError = false): Promise<void> {
```

Comment:
Please add that field in the hook options instead. For example, here we have an option for the timeout. This way you won't have to pass around your variable "throwOnError".

![image](https://user-images.githubusercontent.com/42552874/106957416-f81bc900-6705-11eb-91e8-39dfdc8682d3.png)